framework/src/schema/application_config_schema.ts

```diff
@@ -380,6 +391,9 @@ export const applicationConfigSchema = {
 			port: 8080,
 			host: '127.0.0.1',
 		},
+		backup: {
+			height: 0,
+		},
```

Comment:
Instead of hard-coding it in the application node config, can we not set it dynamically with a command?

This way the migrator can issue the command and the node operators do not need to manually update the node config and restart their nodes before the migration.

Reply:
this is configuration, so command can pass the config dynamically